### ISSUE TITLE
`zetacore` : Register interfaces

### ISSUE THREAD

**Issue by kingpinXD** [zeta-chain/node#3003]
Check if we need to register interfaces twice

We manually register them here 
https://github.com/zeta-chain/zeta-node/blob/c8c833f66e3862cfa9e12c357436322e4f12c807/app/encoding.go#L32-L49

But they also registered the gt with cosmos SDK in the makeconfig call.

**Comment by lumtis** [comment]
>We manually register them here
https://github.com/zeta-chain/zeta-node/blob/c8c833f66e3862cfa9e12c357436322e4f12c807/app/encoding.go#L32-L49

This link doesn't return to an existing code

Why is it needed to register the interface twice?

**Comment by kingpinXD** [comment]
I dont think we do , noticed that it was being done twice while working on a pr , will investigate and remove in a future task
